ReactWindows/ReactNative.Net46/Shell/MainReactPackage.cs

```diff
@@ -0,0 +1,112 @@
+﻿using ReactNative.Animated;
+using ReactNative.Bridge;
+using ReactNative.Modules.AppState;
+using ReactNative.Modules.Clipboard;
+using ReactNative.Modules.Core;
+using ReactNative.Modules.Dialog;
+//using ReactNative.Modules.Image;
+using ReactNative.Modules.I18N;
+//using ReactNative.Modules.Launch;
+//using ReactNative.Modules.Location;
+using ReactNative.Modules.NetInfo;
+using ReactNative.Modules.Network;
+//using ReactNative.Modules.StatusBar;
+using ReactNative.Modules.Storage;
+//using ReactNative.Modules.Vibration;
+using ReactNative.Modules.WebSocket;
+using ReactNative.UIManager;
+using ReactNative.Views.Text;
+using ReactNative.Views.View;
+//using ReactNative.Views.Flip;
+//using ReactNative.Views.Image;
+//using ReactNative.Views.Picker;
+//using ReactNative.Views.Progress;
```

Comment:
Don't we want to leave them all commented out so that we can just uncomment them as they come online?